Sources/HaCWebsiteLib/Views/Workshops/WorkshopsIndexPage/WorkshopsIndexSchedule.swift

```diff
@@ -0,0 +1,49 @@
+import HaCTML
+
+/// Above-the-fold for the WorkshopsIndexPage
+struct WorkshopsIndexSchedule: Nodeable {
+  var node: Node {
+    return El.Div[Attr.className => "WorkshopsIndexPage__schedule WorkshopsIndexSchedule"].containing(
+      feature,
+      upcoming
+    )
+  }
+
+  var feature: Node {
+    return El.Section[Attr.className => "WorkshopsIndexFeature WorkshopsIndexSchedule__feature"].containing(
+      El.H1[Attr.className => "Text--sectionHeading"].containing("Featured"),
+      El.Div[Attr.className => "WorkshopsIndexFeature__hero"].containing(LandingFeatures.programmingInRust)
+    )
+  }
+
+  var upcoming: Node {
+    return El.Div[Attr.className => "WorkshopsIndexSchedule__upcoming WorkshopsIndexUpcoming"].containing(
+      El.Div[Attr.className => "WorkshopsIndexUpcoming__list"].containing(
+        El.A[Attr.href => "#", Attr.className => "WorkshopsIndexUpcomingWorkshop"].containing(
+          El.Span[Attr.className => "WorkshopsIndexUpcomingWorkshop__date"].containing(
+            "23 Jan"
```

Comment:
Are we planning on exposing workshops like we do events (with ones upcoming showing up first)? I assumed discovery would be more content-based.

This discussion doesn't need to block this merging, but we should probably have it before workshops are public

Reply:
Right now he primary use case of our workshops is live viewing so focusing on upcoming is good at the moment I think. 
This page currently doesn’t have an archive section which will be the next step but not necessary for launch imo. 
Archive should include search feature based on the tags. We should probably show popular workshops and allow this search rather than trying to categorise, that could get messy

Reply:
I figured events on the home page would surface upcoming workshops better than a dedicated workshops page could, given it's one less click.

Reply:
We could, for example, have this page be an alphabetised listing (or sorted by date updated) of all workshops until we get some kind of search going

Reply:
I figure it’s useful to have an upcoming workshops list somewhere and this is as good a place as any.

We can discuss archive specifics later. Agreed that workshops belong in the homepage too and will mostly be discovered there.

Reply:
Yep let's bench this for now